packages/nextjs/src/components/_common/relatedSituations/RelatedSituations.tsx

```diff
@@ -28,23 +28,24 @@ export const getAnchorId = (test: string) => {
 
 export const RelatedSituations = ({ relatedSituations, title, description }: Props) => {
     const { language, editorView, page } = usePageContentProps();
-
     const getStringPart = translator('related', language);
     const defaultTitle = getStringPart('otherOffers');
     const actualTitle = (title || defaultTitle).trim(); //Redaktører legger inn et mellomrom hvis de ikke vil ha tittel
+    const actualDescription = description || getStringPart('moreInformation');
 
     return (
-        <div
+        <nav
```

Comment:
Jeg er litt usikker på om det gir mening å endre til `<nav>` for denne.

Reply:
La oss diskutere. Både nav og section er OK, antageligvis.

Reply:
Beslutning: Section istedendfor nav